shared/common-adapters/image.native.js

```diff
@@ -1,9 +1,16 @@
 // @flow
 import * as React from 'react'
 import {NativeImage} from './mobile.native'
-import type {Props} from './image'
+import type {Props, ReqProps} from './image'
 
-export default ({src, style}: Props) => {
+const FileImage = ({src, style}: Props) => {
   const source = {uri: `file://${src}`}
-  return <NativeImage source={source} style={{resizeMode: 'contain', ...style}} />
+  return <NativeImage source={source} style={[{resizeMode: 'contain'}, style]} />
 }
+
+const RequireImage = ({src, style}: ReqProps) => (
```

Comment:
this is an image you can use if you're passing require('myfile.png'). prefixing it with file:// like the old default doesn't work